src/features/library/class-block-content-filter.php

```diff
@@ -48,14 +50,23 @@ public function boot(): void {
 	 * @param string $content Content of the current post.
 	 * @return string Updated content.
 	 */
-	public function filter_the_content( $content ) {
+	public function filter_the_content( string $content ): string {
 		$post = get_post();
 
 		// Skip if 'the_content' is running on non-block content or on content other than the post being viewed.
-		if ( $post && is_single( $post->ID ) && has_blocks( $content ) ) {
+		if (
+			$post
+			&& is_callable( $this->block_merge )
```

Comment:
We should add `callable` to the property type instead.

Reply:
It's already a callable. Not sure why I added this here. 🤔